packages/manager-components/jest.setup.ts

```diff
@@ -0,0 +1,3 @@
+import React from 'react';
+
+global.React = React;
```

Comment:
Why do you put React in global ? Is it a new best practice for jest tests ?

Reply:
Cause in spec file, we didn't use import of react